### ISSUE TITLE
Any plans for an advisor?

### ISSUE THREAD

**Issue by michelmilezzi** [dalibo/temboard#469]
Hi folks,

I've been using Temboard recently and is a great tool. However, It would be interesting if it had an advisor...
A long time ago I made one in Java, based on the rules defined on Greg Smith's book "PostgreSQL 9.0 High Performance". You can check it [here](https://github.com/michelmilezzi/yapott) (although the repository is recent the tool is quite old, supporting only PostgreSQL from 8.3 to 9.5).
Basically, it collects info about the server environment and suggests improvements such as increasing _shared_buffers_ and so on.

Do you have any plans to do a similar feature on Temboard?

**Comment by julmon** [comment]
Hi,

No plan for the moment for a such feature, but I think this is a good idea and we should think about this.
Thanks for the idea, I'm keeping this issue open for a while.
